### ISSUE TITLE
CSRF fails in some situations

### ISSUE THREAD

**Issue by powerpbx** [playsms/playsms#645]
Because CSRF is checked in init.php it will fail sometimes when the page is reloaded causing `$_SESSION['X-CSRF-Token']` to be `NULL`, which then fails the token validation.  I think there are also some situations where there is a mismatch.  So the session token is using the previous `$_POST['X-CSRF-Token']`.  I think that also happens because of some browser extensions or maybe it depends on what url the page is loaded from.

I can usually load it in incognito mode and it usually works again.  It seems to occur when you are accessing by domain name instead of IP.  That could again be due to the way some browser extensions work.

In summary, to reproduce, load the page using a public sub/domain name assigned to the IP instead directly by IP address.  I saw it happen in firefox and chrome. I use several browser extensions but I suspect it may be related to uBlock Origin.
